client/apollo/react/src/Heading/HeadingWithSubheadings.tsx

```diff
@@ -0,0 +1,30 @@
+import type { JSX, ReactNode } from "react";
+import { HeadingLevel } from "./types";
+
+type HeadingTextProps = {
+  title: ReactNode;
+  firstSubtitle?: string;
+  secondSubtitle?: string;
+  level?: HeadingLevel;
+};
+
+export const HeadingWithSubheadings = ({
+  title,
+  firstSubtitle,
+  secondSubtitle,
+  level = 1,
+}: HeadingTextProps) => {
+  const HLevel = `h${level}` as keyof JSX.IntrinsicElements;
+
+  return (
+    <>
+      <HLevel className="af-heading__title">{title}</HLevel>
+      {firstSubtitle && (
+        <span className="af-heading__subtitle">{firstSubtitle}</span>
+      )}
+      {secondSubtitle && (
+        <span className="af-heading__subtitle">{secondSubtitle}</span>
+      )}
+    </>
+  );
+};
```

Comment:
perso je n'aurais pas typé le level avec un union type de number mais `"h1" | "h2"  |  "h3"`
ce qui permet de faire
```suggestion
export const HeadingWithSubheadings = ({
  title,
  firstSubtitle,
  secondSubtitle,
  level: Component = "h1",
}: HeadingTextProps) => {
  return (
    <>
      <Component  className="af-heading__title">{title}</Component>
      {firstSubtitle && (
        <span className="af-heading__subtitle">{firstSubtitle}</span>
      )}
      {secondSubtitle && (
        <span className="af-heading__subtitle">{secondSubtitle}</span>
      )}
    </>
  );
};
```